client/src/components/mixins/receipt-mixin.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component
+export default class ReceiptMixin extends Vue {
+  /** Returns the line item description mapped to a user-friendly name. */
+  rcptDescToName (description: string): string {
+    switch (description) {
+      case 'Reg. Submission Online': return 'Name Request'
+      case 'Upgrade to Priority': return 'Priority Request'
```

Comment:
Are these passed from the back end?

Reply:
Yes. It's part of the SBC payment line items that the BE (either name or sbc-pay, I'm not sure) has prepared.

Reply:
Copy that!